src/Features/CSharp/Portable/ReplaceMethodWithProperty/CSharpReplaceMethodWithPropertyService.cs

```diff
@@ -376,11 +376,24 @@ public void ReplaceSetReference(SyntaxEditor editor, SyntaxToken nameToken, stri
                 return;
             }
 
-            var newName = nameChanged
-                ? SyntaxFactory.IdentifierName(SyntaxFactory.Identifier(propertyName).WithTriviaFrom(nameToken))
-                : nameNode;
-
             var invocation = nameNode?.FirstAncestorOrSelf<InvocationExpressionSyntax>();
+
+            var newName = nameNode;
+            if (nameChanged)
+            {
+                if (invocation == null)
+                {
+                    newName = SyntaxFactory.IdentifierName(SyntaxFactory.Identifier(propertyName)
+                        .WithTriviaFrom(nameToken));
+                }
+                else
+                {
+                    newName = SyntaxFactory.IdentifierName(SyntaxFactory.Identifier(propertyName)
+                        .WithLeadingTrivia(nameToken.LeadingTrivia)
+                        .WithTrailingTrivia(invocation.ArgumentList?.CloseParenToken.TrailingTrivia));
```

Comment:
i don't believe ArgumentList is optional in a C# invocation list.    Do i don't think you need `?.`

Reply:
Done.